djangocms_pageadmin/admin.py

```diff
@@ -332,7 +333,7 @@ def duplicate_view(self, request, object_id):
 
                 placeholders = obj.get_placeholders()
                 for source_placeholder in placeholders:
-                    target_placeholder = new_page_content.placeholders.get(
+                    target_placeholder, created = new_page_content.placeholders.get_or_create(
```

Comment:
We shouldn't need to do this, the copy should have created new placeholders. The cms new and copy is an example of this: 

https://github.com/divio/django-cms/blob/release/4.0.x/cms/models/titlemodels.py#L133

Reply:
Imagine that we have Pagecontent that have a certain set of placeholders. It has a placeholder with the slot named "redundant" that is no longer used. The template for that placeholder has been removed so when we `rescan_placeholder` for new_page_content that placeholder is not created. 

In this scenario we can't rescan placeholders and iterate through all placeholders and use `get` like we do above, because it will still fail.

Reply:
This is ok to use get_or_create due to keeping legacy content rather than ignoring it wih regards to placeholders that may not be picked up by rescan placeholders.